app/controllers/miq_policy_rsop_controller.rb

```diff
@@ -80,18 +80,18 @@ def rsop
   def rsop_option_changed
     assert_privileges('policy_simulation')
     if params[:event_typ]
-      @sb[:rsop][:event] = params[:event_typ] == "<Choose>" ? nil : params[:event_typ]
+      @sb[:rsop][:event] = params[:event_typ] == "" ? nil : params[:event_typ]
```

Comment:
This pattern can be simplified with the `.presence` method.

```suggestion
      @sb[:rsop][:event] = params[:event_typ].presence
```